openshift-client/src/main/java/io/fabric8/openshift/client/DefaultOpenShiftClient.java

```diff
@@ -311,102 +325,102 @@ public KubernetesListMixedOperation lists() {
 
   @Override
   public MixedOperation<Build, BuildList, DoneableBuild, BuildResource<Build, DoneableBuild, String, LogWatch>> builds() {
-    return new BuildOperationsImpl(httpClient, OpenShiftConfig.wrap(getConfiguration()), getNamespace());
+    return new BuildOperationsImpl(httpClient, OpenShiftConfig.withApiGroup(this, BUILD, OpenShiftConfig.wrap(getConfiguration())), getNamespace());
```

Comment:
just starting that - turns out to be a fairly massive change as the code needs the OpenShiftClient so that means all constructors for all operations + all the handlers need to change....